lib/chef/knife/bmcs_server_delete.rb

```diff
@@ -29,16 +29,38 @@ class BmcsServerDelete < Knife
              long: '--wait SECONDS',
              description: 'Wait for the instance to be terminated. 0=infinite'
 
+      option :prune,
+             long: '--prune',
+             description: 'Also remove node from Chef server.'
+
+      option :chef_node_name,
+             long: '--chef-node-name NAME',
```

Comment:
I would mirror the definition from server create: 
      option :chef_node_name,
             short: '-N NAME',
             long: '--node-name NAME',